src/api/SFA.DAS.Provider.Events.Infrastructure/Data/DcfsSubmissionEventsRepository.cs

```diff
@@ -72,6 +75,22 @@ public DcfsSubmissionEventsRepository()
                 .ConfigureAwait(false);
         }
 
+        public async Task<IEnumerable<SubmissionEventEntity>> GetLatestLearnerEventByStandard(long eventId, long uln)

```

Comment:
Just a question, if we are only expecting to return a single record. Why are we returning this a list which will only have one record or none?

Reply:
Good question, looks like the IEnumerable is a hangover to when we ere going to return many results.  Fixing up.

Reply:
Double checked and the query can indeed return more than one result if the learner has more than one standard code in the Submission events.  The rownumber = 1 is just limiting the results to one per standard code.

Reply:
OK by me